design/proposal-to-add-boot-parameters-for-metal3-schema.md

```diff
@@ -0,0 +1,135 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# Proposal to add BOOT parameter in the metal3 schema
+
+## Table of Contents
+
+<!--ts-->
+   * [Proposal to add BOOT parameter in the metal3 schema](#title)
+      * [Table of Contents](#table-of-contents)
+      * [Summary](#summary)
+      * [Motivation](#motivation)
+         * [Goals](#goals)
+      * [Proposal](#proposal)
+         * [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+         * [Risks and Mitigations](#risks-and-mitigations)
+      * [Design Details](#design-details)
+         * [Work Items](#work-items)
+         * [Dependencies](#dependencies)
+         * [Test Plan](#test-plan)
+      * [References](#references)
+<!--te-->
+
+## Summary
+
+Enhancing the `metal3` schema by adding `BOOT` parameters that we received 
```

Comment:
This would be a lot clearer if it said something like "Enhance the `HardwareDetails` section of the BaremetalHost schema". metal3 is a namespace, not a Custom Resource type.

Reply:
Okay.

Reply:
ok